Santouka/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Santouka</title>
+
+  <link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+  <header>
```

Comment:
my approach is

// tag = class
header = main-header
ul = nav-items
li = nav-item
a = nav-item__link
a active = nav-item__link--active

Reply:
i always follow BEM style for css naming

Reply:
I fixed it!